packages/react-components/babel-preset-global-context/src/transformPlugin.ts

```diff
@@ -0,0 +1,259 @@
+            // Adds import for global context
+            path.unshiftContainer('body', createGlobalContextImportDeclaration(GLOBAL_CONTEXT_PACKAGE, false));
+          }
+
+          if (state.importDeclarationPaths.some(hasContextSelectorImport)) {
+            // Adds import for global context selector
+            path.unshiftContainer('body', createGlobalContextImportDeclaration(GLOBAL_CONTEXT_SELECTOR_PACKAGE, true));
+          }
+
+          const packageJson: PackageJSON = JSON.parse(readFileSync(packageJsonPath).toString());
+          // substitutes expressions of react createContext to global context
+          for (const expressionPath of state.contextSelectorExpressionPaths) {
+            expressionPath.replaceWith(
+              createGlobalContextCallExpression({
+                expressionPath,
+                packageJson,
+                packageJsonPath,
+                filePath: state.filename,
+                functionName: GLOBAL_CONTEXT_SELECTOR_ALIAS,
+              }),
+            );
+          }
+
+          for (const expressionPath of state.nativeExpressionPaths) {
+            expressionPath.replaceWith(
+              createGlobalContextCallExpression({
+                expressionPath,
+                packageJson,
+                packageJsonPath,
+                filePath: state.filename,
+                functionName: GLOBAL_CONTEXT_ALIAS,
+              }),
+            );
+          }
+        },
+      },
+
+      /**
+       * Finds imports of `react` or `@fluentui/react-context-selector`.
+       * If `createContext` is explicitly imported, stores its imported name in state
+       *
+       * @example import { createContext as createMyContext } from 'react';
+       */
+      // eslint-disable-next-line @typescript-eslint/naming-convention
+      ImportDeclaration(path, state) {
+        if (!hasReactImport(path) && !hasContextSelectorImport(path)) {
+          return;
+        }
+
+        const native = hasReactImport(path);
+        state.importDeclarationPaths!.push(path);
+
+        for (const importSpecifier of path.node.specifiers) {
+          if (
+            types.isImportSpecifier(importSpecifier) &&
+            types.isIdentifier(importSpecifier.imported) &&
+            types.isIdentifier(importSpecifier.local) &&
+            importSpecifier.imported.name === CREATE_CONTEXT_CALL
+          ) {
+            const localName = importSpecifier.local.name;
+            if (native) {
+              state.nativeLocalName = localName;
+            } else {
+              state.contextSelectorLocalName = localName;
+            }
+          }
+        }
+      },
+      /**
+       * Handles case when `createContext()` is `CallExpression`.
+       *
+       * @example createContext({})
+       */
+      // eslint-disable-next-line @typescript-eslint/naming-convention
+      CallExpression(path, state) {
+        if (state.importDeclarationPaths!.length === 0) {
```

Comment:
can we stop using TS deprecated non null assertion syntax and instead use standard JS ?

```suggestion
        if (state.importDeclarationPaths?.length === 0) {
```